bentoml/deployment/serverless.py

```diff
@@ -92,3 +99,31 @@ def deploy_with_serverless(platform, archive_path, extra_args):
     output_path = generate_serverless_bundle(bento_service, platform, archive_path, extra_args)
     subprocess.call(['serverless', 'deploy'], cwd=output_path)
     return output_path
+
+
+def delete_serverless_deployment(platform, archive, region, stage):
+    bento_server = load(archive)
+    if platform == 'google-python':
+        config = {
+            "service": bento_server.name,
+            "provider": {
+                "name": "google",
+                "region": DEFAULT_GCP_REGION if region is None else region,
+                "stage": DEFAULT_GCP_DEPLOY_STAGE if stage is None else stage
+            }
+        }
+    elif platform == 'aws-lambda' or platform == 'aws-lambda-py2':
+        config = {
+            "service": bento_server.name,
+            "provider": {
+                "name": "aws",
+                "region": DEFAULT_AWS_REGION if region is None else region,
+                "stage": DEFAULT_AWS_DEPLOY_STAGE if stage is None else stage
+            }
+        }
+    yaml = YAML()
+    with TempDirectory() as tempdir:
+        saved_path = os.path.join(tempdir, 'serverless.yml')
+        yaml.dump(config, Path(saved_path))
+        subprocess.call(['serverless', 'remove'], cwd=tempdir)
```

Comment:
let's create a serverless class for accessing these commands? that way we can put the logic of checking the output to determine command status in there. And it makes it much easier to mock and add tests.

Reply:
yep

Reply:
done